python/google/protobuf/message.py

```diff
@@ -182,7 +182,7 @@ def ParseFromString(self, serialized):
     do not return the value that MergeFromString returns.
```

Comment:
This docstring is now out of date

Reply:
Fixed. Thanks.

I see now that the Python API documentation at https://developers.google.com/protocol-buffers/docs/reference/python/google.protobuf.message.Message-class is out of date (from an older version of protobuf that didn't have this text), although I don't see anywhere to report or suggest updates to the documentation site.